kkeclipse/WebContent/WEB-INF/jsp/CatalogCheckoutFinishedBody.jsp

```diff
@@ -59,3 +59,52 @@
 						</form>
 			    	</div>
 	    		</div>
+	    		
+	    		<%if (kkEng.getCustomerMgr().getCurrentCustomer() != null && kkEng.getCustomerMgr().getCurrentCustomer().getType() != 2 && kkEng.getCustomerMgr().getCurrentCustomer().getGlobalProdNotifier() == 0) { %>
+	    		
+	    		 <div
+
+  id="refcandy-mint"
+
+  data-app-id="ijdpgu342un7laa7kzhijptcs"
+
+  data-fname="<%= kkEng.getCustomerMgr().getCurrentCustomer().firstName %>"
+
```

Comment:
Java is giving syntax error  at this point ('firstName cannot be resolved or is not a field')...maybe you can help me to resolve this error@shalmali